package.json

```diff
@@ -48,20 +48,20 @@
   "homepage": "https://github.com/mojaloop/email-notifier#readme",
   "devDependencies": {
     "chai": "4.2.0",
-    "coveralls": "3.0.5",
+    "coveralls": "3.0.6",
     "faucet": "0.0.1",
-    "istanbul": "1.1.0-alpha.1",
+    "istanbul": "0.4.5",
```

Comment:
please revert version of istanbul to 1.1.0-alpha.1

Reply:
Performed per review.